pkg/apis/experimental/install/install.go

```diff
@@ -51,14 +51,16 @@ func init() {
 		GroupVersion: groupVersion,
```

Comment:
why? registeredGroupVersions is an array of all the groupVersions that has a "experimental/" prefix.